Reference/google/protobuf/unittest_custom_options.pb.swift

```diff
@@ -1684,14 +1906,24 @@ public struct ProtobufUnittest_OldOptionType: ProtobufGeneratedMessage {
 
   }
 
-  public var value: ProtobufUnittest_OldOptionType.TestEnum = ProtobufUnittest_OldOptionType.TestEnum.oldValue
+  private var _value: ProtobufUnittest_OldOptionType.TestEnum? = nil
+  public var value: ProtobufUnittest_OldOptionType.TestEnum {
+    get {return _value ?? ProtobufUnittest_OldOptionType.TestEnum.oldValue}
+    set {_value = newValue}
+  }
+  public var hasValue: Bool {
+    return _value != nil
+  }
+  public mutating func clearValue() {
+    return _value = nil
+  }
 
   public init() {}
 
   public mutating func _protoc_generated_decodeField(setter: inout ProtobufFieldDecoder, protoFieldNumber: Int) throws -> Bool {
     let handled: Bool
     switch protoFieldNumber {
-    case 1: handled = try setter.decodeRequiredField(fieldType: ProtobufUnittest_OldOptionType.TestEnum.self, value: &value)
+    case 1: handled = try setter.decodeOptionalField(fieldType: ProtobufUnittest_OldOptionType.TestEnum.self, value: &_value)
```

Comment:
In `unittest_custom_options.proto`, the field `OldOptionType.value` is a required field.